internal/docs/exported_fields.go

```diff
@@ -70,6 +108,12 @@ func renderFieldsTable(builder *strings.Builder, collected []fieldsTableRecord)
 	if metricTypesPresent {
 		builder.WriteString("---|")
 	}
+	if valuesPresent && includeValues {
+		builder.WriteString("---|")
+	}
+	if examplePresent && includeExamples {
+		builder.WriteString("---|")
+	}
```

Comment:
This is going to add two columns, right? Could you check how this looks in Kibana?

Reply:
While it's still okay in terms of formatting, it might make more sense to combine both in one column, putting the value for when it's a constant keyword and the example in all other cases. WDYT?

![image](https://github.com/user-attachments/assets/35aa0e39-8cdf-4929-a560-6e572e29a114)
![image](https://github.com/user-attachments/assets/ec7dfd73-1d61-44e9-b249-3a77c370ee05)

Reply:
Yes, I think it can be a good idea to put both in the same column.

Reply:
Changed the behaviour so that example gets added by default, unless it's a constant_keyword. In that case, the value is set.

We would require all fields to have the  **example** key set and all fields with type constant_keyword to have the **value** key set for the table to populate nicely. If none of it is available, the column won't be there.

New view:
![image](https://github.com/user-attachments/assets/c7d8b90f-3b42-4448-8516-5638430fbaeb)
![image](https://github.com/user-attachments/assets/0560616f-1853-487d-b846-142e3381bc26)
![image](https://github.com/user-attachments/assets/c4d014a1-7f96-4db5-b1f0-4b347b3dd816)

Reply:
> We would require **all** fields to have the example key set and all fields with type constant_keyword to have the value key set for the table to populate nicely.

Is it ok if some fields are not set, right? They would only have their cells empty, isn't it?

Reply:
@kpollich what do you think about adding this column to this table? Could it cause any problem in Kibana?

Reply:
> > We would require **all** fields to have the example key set and all fields with type constant_keyword to have the value key set for the table to populate nicely.
> 
> Is it ok if some fields are not set, right? They would only have their cells empty, isn't it?

Yes indeed. It would result in an empty cell